devops/docker/ubuntu-14.04-amd64/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get -y update && apt-get -y install \ 
     gcc \
     git \
-    cmake \
+    cmake3 \
```

Comment:
> cmake3 \ [](http://example.com/codeflow?start=3&length=9)

Why? Our requirements for building 'OSConfig for MC' does not require CMake 3, never did.

Reply:
Please see the other reply.

Reply:
We do not believe installing CMake 3 from package is the way here, we'll rather bring back building from source.

Reply:
https://github.com/Azure/azure-osconfig/actions/runs/11490120376/job/31980227024 disagrees with what you're saying:
```
-- The C compiler identification is GNU 4.8.4
-- The CXX compiler identification is GNU 4.8.4
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
CMake Error at CMakeLists.txt:9 (cmake_minimum_required):
  CMake 3.2 or higher is required.  You are running version 2.8.12.2
```

Reply:
https://github.com/Azure/azure-osconfig/actions/runs/11507251502